doc/source/ray-air/deployment.rst

```diff
@@ -0,0 +1,60 @@
+.. _air-deployment:
+
+AIR Deployment Guide
+====================
+
+Here, we describe how you might use or deploy AIR in your infrastructure. There are two main deployment patterns -- a la carte and integration with existing platform.
+
+The core idea is that AIR can be **complementary** to your existing infrastructure and integration tools.
+
+Design Principles
+-----------------
+
+* Ray AIR handles the heavyweight compute aspects of AI apps and services.
+* Ray AIR relies on external integrations (e.g., Tecton, MLFlow, W&B) for Storage and Tracking.
+* Workflow Orchestrators (e.g., AirFlow) are an optional component that can be used for scheduling recurring jobs, launching new Ray clusters for jobs, and running non-Ray compute steps.
+* Lightweight orchestration of task graphs within a single Ray AIR app can be handled with Ray Workflows.
+* Ray AIR libraries can be used independently, within an existing ML platform, or to build a Ray-native ML platform.
+
+
+Ray AIR libraries à la carte
+----------------------------
+
+You can pick and choose which Ray AIR libraries you want to use without replacing all of your ML infrastructure to use Ray AIR.
+
+This is applicable to you if you are an ML engineer wants to independently use a Ray AIR library for a specific AI app or service use case and do not need to integrate with an existing ML platform.
+
+For example, Alice wants to use RLlib to train models for her work project. Bob wants to use Ray Serve to deploy his model pipeline. In both cases, Alice and Bob can leverage these libraries independently without any coordination.
+
+This scenario describes most usages of Ray libraries today.
+
+.. image:: /images/air_arch_1.png
+
+
+Highlighted points
```

Comment:
Bullet points are highlighted points, so that's kind of understood. Can we maybe just lead over to the "highlights" with a quick sentence instead?

Also, I don't feel like these two points will help users much in this context. We don't really know what Autoscaler, Anyscale or KubeRay are at this point (this is all high-level on purpose up to this point). How does pick-and-choose of libraries relate to managed clusters?

I'd also suggest to remove "GCS", unless we want to explain what it is or link to an explanation.

Reply:
Updated!